support/Environments/expanse_gcc.sh

```diff
@@ -5,19 +5,36 @@
 
 # Load system modules
 spectre_load_sys_modules() {
+    # Make sure the core default modules are loaded.
+    # Without these you can get weird compilation errors
+    # that make no sense.
+    module load shared cpu sdsc DefaultModules slurm/expanse numactl
+
     module load gcc/10.2.0
-    module load intel-mkl/2019.1.144
+    module load openblas/0.3.10-openmp
```

Comment:
Why `openblas` vs `intel-mkl`?

Reply:
Intel MKL didn't build and link correctly for me